IBM/carbon-addons-iot-react
# fix(table): column width fix for sortable header

Closes #1381

**Summary**
The column width, i.e. the initially set width of a column obj from the column prop, was not applied to sortable columns. 

**Change List (commits, features, bugs, etc)**
- Made the `initialWidth` an explicit prop of the TableHeader
- Added a width prop to the sortable th in TableHeader
- updated a table story so that this fix could be verified
- updated unit tests

**Acceptance Test (how to verify the PR)**
1. go to [watson-iot-table--with-resize-and-initial-column-widths-on-simple-stateful-with-row-selection-sort](https://deploy-preview-1383--carbon-addons-iot-react.netlify.app/?path=/story/watson-iot-table--with-resize-and-initial-column-widths-on-simple-stateful-with-row-selection-sort)
2. verify that the every other column has width 100px and 200px using the browser debug inspector
